### PR TITLE
this is algorithm of bucket-sort in java language.

### PR DESCRIPTION
**Pull request by Anshuvats**

### CODE REVIEW

File: Bucket-sort/Bucket-sort.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+}
+
+
+/*Output
+Bucket sort in Java
+integer array before sorting
+[80, 50, 30, 10, 90, 60, 0, 70, 40, 20, 50]
+integer array after sorting using bucket sort algorithm
+[0, 10, 20, 30, 40, 50, 50, 60, 70, 80, 90]*/
```
Comment (jainaman224) [inline Bucket-sort/Bucket-sort.java]: Add new line.

**Comment by jainaman224** [comment]
Please fix file names and refer the issue the number.

**Comment by Anshuvats** [comment]
Sure,sir😊

**Comment by Anshuvats** [comment]
#552  issue number.

**Comment by Anshuvats** [comment]
then after what will do?,sir

**Comment by jainaman224** [comment]
Please use more understandable variable names. Then I will help you in rest.
Like file name
~Bucket-sort/Bucket-sort.java~ -> *Bucket_Sort/Bucket_Sort.java*
Remove file Bucket-sort/bucket-sort.java.txt .

**Comment by Anshuvats** [comment]
okay,sir. i will update as soon as possible.

**Comment by Anshuvats** [comment]
thank you sir.

**Pull request merged**